Contents/Code/plex_api_helper.py

```diff
@@ -87,6 +87,35 @@ def setup_plexapi():
     return plex
 
 
+def is_field_locked(item, field_name):
+    # type: (any, str) -> bool
+    """
+    Check if the specified field is locked.
+    
+    This is used to ensure a field has not been manually set by the user.
+
+    Parameters
+    ----------
+    item : any
+        The Plex item to check.
+    field_name : str
+        The name of the field to check.
+
+    Returns
+    -------
+    bool
+        True if the field is locked, False otherwise.
+    """
+
+    Examples
+    --------
+    # add an example please
```

Comment:
```suggestion
    >>> is_field_locked(item=<plex_item>, field_name='summary')
    False
    """
```

Sorry, I didn't complete that in my previous review. If you could add "Examples" to all new functions it would be good. It's for the auto doc in Sphinx.